services/state/schedule/schedule.proto

```diff
@@ -141,12 +141,10 @@ message Schedule {
  * simplified hash code.
  */
 message ScheduleList {
-    /**
-     * a list of schedules, in no particular order.
-     * <p>
-     * While the order is not _specified_, it MUST be deterministic.
-     */
-    repeated Schedule schedules = 1;
+  /**
+   * a list of schedules, in no particular order.
+   */
+  repeated Schedule schedules = 1;
```

Comment:
I would recommend reversing this specific change (change services to match).
The documentation in this repo is more clear, and the indentation is correct.